app/core/models/ocelot/errors/GuidanceErrors.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package models.ocelot.errors
+package core.models.ocelot.errors
```

Comment:
IntelliJ is reporting a typo at line 39: InconsistenQuestionError

Reply:
fixed